mosec/worker.py

```diff
@@ -186,3 +189,30 @@ def forward(self, data: Any) -> Any:
                 will go through ``<deserialize_ipc> -> <forward> -> <serialize_ipc>``
         """
         raise NotImplementedError
+
+    @classmethod
+    def get_forward_json_schema(
+        cls, target: ParseTarget  # pylint: disable=unused-argument
+    ) -> Tuple[Dict[str, Any], Dict[str, Any]]:
+        """Retrieve the JSON schema for the `forward` method of the class.
+
+        Args:
+            cls (type): The class object.
+            target (ParseTarget): The target variable to parse the schema for.
+
+        Returns:
+            Tuple[Dict[str, Any], Dict[str, Any]]: A tuple containing the schema
```

Comment:
No need to add the type annotations here. The sphinx extension will parse it from the method annotations.